### ISSUE TITLE
has a trojan in it do not download 

### ISSUE THREAD

**Issue by OneThousandSorrows** [Pizzabelly/EasyRP#204]
virustotal have found a trojan

**Comment by irismessage** [comment]
Maybe provide a link?

No problem with the linux executable

https://www.virustotal.com/gui/file/f72cb0216286fcee749077d1167a5b55ed00022c7003d4efd4be22354124d8c1

**Comment by irismessage** [comment]
1/70 flagged the windows exe as dangerous. False positive as I've had no issue with it

https://www.virustotal.com/gui/file/78ca6eb3d975bb6bce9efe2556074acf05f3317163251a7dfd804f40f20efd78

**Comment by ghost** [comment]
I agree with joels, likely a false positive. You can always build from source instead of using their executable.
